proposal/first-class-css-functions.md

```diff
@@ -0,0 +1,135 @@
+# First-Class CSS Functions
+
+*([Issue](https://github.com/sass/sass/issues/3176),
+[Changelog](first-class-css-functions.changes.md))*
```

Comment:
I don't think you would need the changelog link on your proposal until the second draft.